qa/TL1_git_submodule/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+test_body() {
+    export TMP_PATH="$(mktemp -d)"
+    cd ..
+    export DALI_REPO_DIR=$(pwd)
+    pushd $TMP_PATH
+    git init
+    git submodule add $DALI_REPO_DIR
+    pushd *
+    git submodule sync --recursive
+    git submodule update --init --recursive
+    popd
+    touch main_stub.cc
+    mkdir build
+    pushd build
+    cmake -D CUDA_TARGET_ARCHS=60 -D CMAKE_BUILD_TYPE=Debug ..
```

Comment:
```suggestion
    cmake -D CMAKE_BUILD_TYPE=Debug ..
```
Better not to assume archs

Reply:
It is just for compilation test, and it shortens the compilation time.